lib/screens/qpi/add_qpi.dart

```diff
@@ -1,29 +1,353 @@
+import 'package:admu_student_app/constants/app_colors.dart';
 import 'package:flutter/material.dart';
 
 import 'package:admu_student_app/models/course.dart';
+import 'package:admu_student_app/widgets/input_field.dart';
+import 'package:admu_student_app/widgets/buttons.dart';
+import 'package:admu_student_app/widgets/select_color.dart';
+import 'package:google_fonts/google_fonts.dart';
 
 class AddQPIPage extends StatefulWidget {
-  final bool isEditing;
+  // final bool isEditing;
 
-  // template, to add a new course
-  final int yearNum;
-  final int semNum;
+  // // template, to add a new course
+  // final int yearNum;
+  // final int semNum;
 
-  // for editing
-  final Course course;
+  // // for editing
+  // final Course course;
 
-  AddQPIPage({this.isEditing, this.yearNum, this.semNum, this.course});
+  // AddQPIPage({this.isEditing, this.yearNum, this.semNum, this.course});
 
   @override
   _AddQPIPageState createState() => _AddQPIPageState();
 }
 
 class _AddQPIPageState extends State<AddQPIPage> {
+  bool yearSelected = true;
+  bool semesterSelected = false;
+  bool courseSelected = false;
+  String title = 'Year';
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(),
-      body: Text('add/edit qpi'),
+      appBar: AppBar(
```

Comment:
use `autoImplyLeading: false` or something similar to change the leading icon to the close icon (x)
it also doesn't have any title in the wireframes, and you can add `actions: [Text()]` for the `Done` at the top-right side